plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/Fabric8FormPage.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat Inc..
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat Incorporated - initial API and implementation
+ *******************************************************************************/
+
+package com.redhat.fabric8analytics.lsp.eclipse.ui;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.m2e.core.project.IMavenProjectChangedListener;
+import org.eclipse.m2e.core.project.MavenProjectChangedEvent;
+import org.eclipse.m2e.editor.pom.MavenPomEditor;
+import org.eclipse.m2e.editor.pom.MavenPomEditorPage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.ui.IEditorPart;
+import org.eclipse.ui.IWorkbenchPage;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.forms.IManagedForm;
+import org.eclipse.ui.forms.editor.FormPage;
+import org.eclipse.ui.forms.widgets.FormToolkit;
+import org.eclipse.ui.forms.widgets.ScrolledForm;
+import org.eclipse.ui.IFileEditorInput; 
+
+import com.redhat.fabric8analytics.lsp.eclipse.ui.internal.TokenCheck;
+
+/**
+ * Class to create contents of page Fabric8Analyses.
+ * 
+ * @author Geetika Batra
+ *
+ */
+class Fabric8FormPage extends MavenPomEditorPage implements IMavenProjectChangedListener  {
+
+	private EditorComposite dependenciesComposite;
+	public Fabric8FormPage(MavenPomEditor pomEditor) {
+		super(pomEditor, "fabric8.analysis ", "Fabric8 Analysis");
+
+	}
+
+	@Override
+	protected void createFormContent(IManagedForm managedForm) {
+		FormToolkit toolkit = managedForm.getToolkit();
+
+	    ScrolledForm form = managedForm.getForm();
+	    form.setText("This page analyses stack of current open project. Click Generate Analyses Report to Proceed ");
```

Comment:
Why is the whole project analyzed? I would supposed that since this is pom.xml file only this file would be analyzed. 
The proceed should be in lowercase.

Reply:
The whole idea of another page was to make user aware that there is a feature called Fabri8Analyses. Instead of displaying browser on Fabric8FormPage, can we show it in Stack view itself? This will remove the mess that will be created if 2 pom.xml's are open. And in the form text we can mention that clicking the button will use all the subsequent manifest files for analyses.

Reply:
No, I like the idea of showing the results in the editor. And I think we could make it work without problems. It is just that I would expect to analyze just the current pom file, not the whole project.

Reply:
I have discussed it with the team. Your solution sounds good to me. Let's stick to analyzing only current pom file in the tab and for project wide analyses, right clicking on Stack Analyses, all the pom files should be considered.